### PR TITLE
Implement multiple label

### PR DESCRIPTION
**Pull request by rafsun42**
Add support for multiple labels in CREATE, MERGE and MATCH clause.

**Important Notes:**

- Before merging, please rerun the workflow again. Any tests that were added after this branch was last updated will fail because this PR makes minor changes in how vertex and edge objects are output.

**Update:**
- Base of this PR is changed to a new remote branch `multiple_label_16`, which is based off `master`.

### CODE REVIEW

File: src/backend/parser/cypher_label_expr.c [hunk 1]
```diff
@@ -0,0 +1,656 @@
+#include "postgres.h"
```
[the remark this answers is not in the document]
Comment (rafsun42) [reply src/backend/parser/cypher_label_expr.c:1]: Good catch!

File: src/backend/parser/cypher_label_expr.c [hunk 2]
```diff
@@ -0,0 +1,656 @@
...
+    {
+        /*
+         * generates a name for intersection relation
+         * i.e. for CREATE (:A:B:C), _agr_ABC
+         */
+        StringInfo relname_strinfo;
+        ListCell *lc;
+        char *relname;
+
+        Assert(list_length(label_expr->label_names) > 1);
```
[the remark this answers is not in the document]
Comment (rafsun42) [reply src/backend/parser/cypher_label_expr.c:392]: Ah! I remember you reported this one before. I will fix it.

**Comment by dehowef** [comment]
@rafsun42 Looks good to me-- this seems to be a really big change seeing that it affects the regression tests in a large way. I think it should be a priority to get this reviewed by as many  people as possible, so that development can continue. @jrgemignani mind taking a look?

File: src/backend/parser/cypher_label_expr.c (hunk 2 above)
[the remark this answers is not in the document]
Comment (rafsun42) [reply src/backend/parser/cypher_label_expr.c:392]: Added this in my backlog.

**Comment by rafsun42** [comment]
@jrgemignani @MuhammadTahaNaveed This PR is ready for review now. It was rebased to a new remote branch `multiple_label_16`, which is based off master. So, there won't be any merge conflicts when changes are added to master. Once it is merged, the remote `multiple_label_16` needs to be rebased and merged to master.

**Comment by rafsun42** [comment]
Moved to #2082

**Pull request closed without merging**